### DISCUSSION TITLE
Is skia-canvas production ready now?

### DISCUSSION THREAD

**Discussion by xeoshow** [samizdatco/skia-canvas#217] [General]
Hello,

I am new to skia-canvas, just would like to know if the latest version is production ready now?
Thanks.

Best regards, Jason

**Comment by eric-gitta-moore** [comment]
+1
any differences with https://github.com/Brooooooklyn/canvas ?

**Comment by Santas** [comment]
We use skia-canvas in production to generate ±17M images/day without any issues.
